src/projects/create-pull-request/create-pull-request.html

```diff
@@ -91,6 +91,23 @@
       on-response="_showPullRequestInformation"
       ></github-authenticated-ajax>
 
+      <iron-ajax
+        id="createPull"
+        method="POST"
+        url="[[backendPathCreate]]"
+        headers='{"Content-Type": "application/json"}'
+        handle-as="json"
+        last-response="{{number}}"
+        on-response="_setStatus"
+        ></iron-ajax>
+
+      <iron-ajax
+        id="setPrStatusafterCreate"
+        method="POST"
+        url="[[backendPathStatus]]"
+        headers='{"Content-Type": "application/json"}'
+        handle-as="json"></iron-ajax>
```

Comment:
This element seems to be duplicate in `pr-status.html`. Can we maybe reuse that element for setting the status?